### DISCUSSION TITLE
希望pcl能支持分支forge加载器：NeoForge

### DISCUSSION THREAD

**Discussion by ghost** [Meloong-Git/PCL#3739] [讨论]
### 描述
hmcl和BakaXL都以支持neoforge模组加载器，而且在1.20.x高版本越来越多的mod开始支持neoforege还有的mod已经放弃 Forge转入neo Forge的开发当中，比如AoA3 已移植到 [NeoForge]，不再支持 Forge。所以我也希望pcl能够加入neoforge的安装方式。
### 原因
此加载器相比较forge有着更好的前景。

**Comment by allMagicNB** [comment]
![Screenshot_20240426_205727.jpg](https://github.com/Hex-Dragon/PCL2/assets/116283787/d3e4f1d6-db19-467e-a785-8b4ef4eef44b)

**Comment by MoYuan-CN** [comment]
#2244

**Comment by Griseo-bh3rd** [comment]
~不是就算没看常见列表这玩意就在issue置顶啊~
